src/main/java/core/basesyntax/Application.java

```diff
@@ -1,8 +1,13 @@
 package core.basesyntax;
 
-public class Application {
+public class Application implements Cloneable {
```

Comment:
```suggestion
public class Application {
```
you implement this interface if you override clone(). In your case you don't clone Application class

Reply:
Sorry, forgot to remove it